lang/hr.js

```diff
@@ -150,7 +156,7 @@ var language = {
       },
       "command-flash": {
         "title": "Učitaj (\"flash\") projekt izravno u micro:bit",
-        "label": "Flash"
+        "label": "Flash ('flashaj')"
```

Comment:
Not sure if this is really better, is just `Flashaj` not acceptable? The answer could be "no", but we should ask.

Reply:
I'm not sure, the response I got was "Please leave the quotation marks, as this signifies it's an English word we have co-opted into the Croatian language"

Reply:
I think in that case I'd rather have `'Flashaj'` than `Flash ('flashaj')`. Either way, could we explain that this is for the buttons, and if it would be okay to have the word without the quotation marks only in this scenario?

Reply:
I have requested this change in crowdin https://crowdin.com/translate/microbitorg/3298/en-hr#q=flash